src/logger/sentry/setup/index.ts

```diff
@@ -27,5 +27,15 @@ init({
   environment: process.env.NODE_ENV,
   dist,
   release,
-  ignoreErrors: [`t is not defined`, `Can't find variable: t`],
+  ignoreErrors: [
+    /*
+     * Unknown internals errors
+     */
+    `t is not defined`,
+    `Can't find variable: t`,
+    /*
+     * Un-useful errors
+     */
+    `Network request failed`,
```

Comment:
We got ~100k of these from throughout the app, just generic network failures. We should probably be better about gating errors using `isNetworkError`, but this should be safe. Any spikes in error rates should be detected in DataDog, and we should be able to safely ignore client connections e.g. bc of flakiness.